homeassistant/components/lametric/config_flow.py

```diff
@@ -255,7 +255,7 @@ async def _async_step_create_entry(self, host: str, api_key: str) -> FlowResult:
                 model=Model(
                     cycles=2,
                     frames=[Simple(text="Connected to Home Assistant!", icon=7956)],
-                    sound=Sound(id=NotificationSound.WIN),
+                    sound=Sound(sound=NotificationSound.WIN),
```

Comment:
`id` is only an alias for `sound`. Furthermore,  the model config sets `allow_population_by_field_name = True` which causes the `__init__` method to be synthesized with the actual field name and not the alias, for type checking (see: https://docs.pydantic.dev/mypy_plugin/#generate-a-signature-for-model__init__).

https://github.com/frenck/python-demetriek/blob/v0.4.0/src/demetriek/models.py#L134
https://github.com/frenck/python-demetriek/blob/v0.4.0/src/demetriek/models.py#L159-L162

Reply:
> id is only an alias for sound.

Yup, current worked with mypy :)
This is better of course 👍